Dockerfile

```diff
@@ -14,8 +17,8 @@ COPY / /usr/picard/
 WORKDIR /usr/picard
 
 # Build the distribution jar, clean up everything else
-RUN ./gradlew shadowJar && \
-    mv build/libs/picard.jar picard.jar && \
+RUN ./gradlew ${build_command} && \
+    mv build/libs/${jar_name} ${jar_name} && \
```

Comment:
Oh! Last thing, could we always name the target `picard.jar`? That way you can switch the docker in and out without needing to edit your commands.